white-test-app-backend/src/main/java/pl/lodz/p/white/whitetestapp/accountmanager/dtos/AccountDto.java

```diff
@@ -0,0 +1,40 @@
+package pl.lodz.p.white.whitetestapp.accountmanager.dtos;
+
+public class AccountDto {
+    private String username;
+    private String email;
+    private String password;
+    private String lang;
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
```

Comment:
gettery i settery mogą zwracać `this`. zobacz inne obiekty dto/requesty/responsy albo enitity